### DISCUSSION TITLE
PML Playground

### DISCUSSION THREAD

**Discussion by tajmone** [pml-lang/pml-companion#28] [Show and tell]
@pml-lang and @celtic-coder, I've just created __The PML Playground__, a new repository dedicated to general experiments with PML:

- https://github.com/tajmone/pml-playground

I've invited both of you as collaborators, so you'll have freedom to add your own sub-projects there. The idea is to host various proof-of-concept tools, prototypes and assets, organized into independent folders. So you have total freedom to push your contents directly to `main` branch.

Each collaborator is the maintainer of his own assets, so changes to someone else's assets should go through the usual PR and review process, as if each sub-project/asset was an independent repository (although it's actually just a folder or set of files).

I'll be using this repo for some ideas I wanted to test, like pandoc filters (#26), re-implementing the default CSS stylesheets in Sass, etc.

At this stage, having a single and commonly shared repository to experiment with idea seems more sensible than creating multiple individual repositories or relying on GitHub Gists. When a sub-projects reaches maturity it can always be moved to an independent repository. In the meanwhile, sharing assets in a common pool should render all work easier.

**Comment by celtic-coder** [comment]
Thanks, Tristano (@tajmone ), an excellent idea!

**Reply by pml-lang** [reply]
Yes, excellent idea!
Thank you, Tristano.
